model/riscv_pmp_control.sail

```diff
@@ -48,14 +48,43 @@ function pmpCheckRWX(ent, acc) = {
   }
 }
 
-// this needs to be called with the effective current privilege.
+/* this needs to be called with the effective current privilege */
```

Comment:
`//` style comments are nicer IMO. I don't see why we would change them back?

Reply:
This is the part that was changed by @HamzaKh01, I didn't check this as I was too focused on the code implementation part, we can change it back if needed, but there are many one line comments like `/* ... */` in Sail, we might need to declare it better in the codestyle after some discussion?

Reply:
Yeah there is an experimental auto-formatter in the Sail compiler but I don't know what state it got to and I'd guess it doesn't change this sort of thing.